packages/terra-doc-template/src/terra-dev-site/doc/example/BadgeExamples.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved, import/extensions
+import NpmBadge from '../../../NpmBadge';
+import { name } from '../../../../package.json';
+
+const packageUrl = 'https://engineering.cerner.com/terra-ui/components/terra-doc-template/doc-template/doc-template';
+const packageVersion = '2.13.0';
```

Comment:
For this file, let's actually pull the version from the package.json file.

```jsx
import { name, version } from '../../../../package.json';
```
and then pass `version` to the `packageVersion` prop. For the tests files, we can keep the version static.